src/core/echarts.ts

```diff
@@ -428,6 +428,15 @@ class ECharts extends Eventful<ECEventDefinition> {
                 : devUseDirtyRect;
         }
 
+        zrender.registerSSRDataGetter(el => {
+            const ecData = getECData(el);
+            const hashMap = createHashMap();
+            hashMap.set('series_index', ecData.seriesIndex);
```

Comment:
I will suggest returning an object in this method to ensure it will be typesafe. To avoid the situation that we change the key here and forget to change where reads it.

Reply:
Do you mean we should define the type in ZRender? But ZRender should not know the structures like seriesIndex.

Reply:
We can use generic type
```ts
zrender. registerSSRDataGetter<Data>(el => ...)
```
Just like what `getECData` do